src/common/utils.lua

```diff
@@ -196,34 +196,26 @@ function utils.reply(msg)
 	Handlers.utils.reply(msg)
 end
 
-function utils.validateArweaveId(id)
-	local valid = string.match(id, constants.ARWEAVE_ID_REGEXP) == nil
-
-	assert(valid == true, constants.INVALID_ARWEAVE_ID_MESSAGE)
-end
-
-function utils.validateOwner(caller)
-	local isOwner = false
-	if Owner == caller or Balances[caller] or ao.env.Process.Id == caller then
-		isOwner = true
+function utils.parseAntState(data)
+	assert(type(data) == "string", "Data must be a string")
+	local decoded = json.decode(data)
```

Comment:
Only records and balances benefit from having a local version here. Might as well save lines of code and just excise all of them.

Reply:
Can we actually just excise all the fields we don't even need? e.g.
```
return {
  Owner: decoded.Owner
  Controllers: <mapping of decoded.Controllers array to the format we want to work with internally>
}
```